scripts/qa_crowbarsetup.sh

```diff
@@ -2615,12 +2619,12 @@ function custom_configuration
                 # this should be adapted when NFS mode is supported for data cluster
                 proposal_set_value ceilometer default "['attributes']['ceilometer']['use_mongodb']" "false"
 
-                local ceilometernodes=($unclustered_nodes)
+                local ceilometernodes=(${unclustered_nodes[@]})
                 # make sure we do not pick SP1 nodes on cloud7
                 if [ -n "$deployceph" ] && iscloudver 7 ; then
-                    ceilometernodes=$unclustered_sles12plusnodes
+                    ceilometernodes=${unclustered_sles12plusnodes[@]}
                 fi
-                if [[ ! $ceilometernodes ]]; then
+                if [[ ${#ceilometernodes} -eq 0 ]]; then
```

Comment:
This check has the same drawback as `[[ ! $variable ]]`, it fails for `variable=('' a b c)`. It will report it as empty array because first element is empty. The correct way to get number of elements in array is `${#variable[@]}`.